driver/syscall_ia32_64_map.c

```diff
@@ -0,0 +1,363 @@
+/*
+ * This file was automatically created by syscalls-bumper (https://github.com/falcosecurity/syscalls-bumper).
+ * DO NOT EDIT THIS FILE MANUALLY.
+ */
+
+#include "ppm_events_public.h"
+
+const int g_ia32_64_map[SYSCALL_TABLE_SIZE] = {
```

Comment:
This file contains the mapping between a 32bit syscall number (array index) and its 64bit value.

Reply:
It is autogenerated by syscalls-bumper; i can add a comment explaining what the map does ;)

Reply:
Added the comment.

Reply:
Also, as discussed privately with @Andreagit97 , in the future we might push a generic `syscall32` event when the 32bit syscalls have no 64bit mapping (the `-1` values in the array).
Or we could push a `syscall` generic event adding a new `bool is32bit` parameter.
I'll add this to the PR body future work.